tempodb/backend/instrumentation/backend_transports.go

```diff
@@ -0,0 +1,80 @@
+package instrumentation
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	gcsRequestDuration = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "tempodb",
+		Name:      "gcs_request_duration_seconds",
+		Help:      "Time spent doing GCS requests.",
+		Buckets:   prometheus.ExponentialBuckets(0.005, 4, 6),
+	}, []string{"operation", "status_code"})
+
+	s3RequestDuration = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "tempodb",
+		Name:      "s3_request_duration_seconds",
+		Help:      "Time spent doing AWS S3 requests.",
+		Buckets:   prometheus.ExponentialBuckets(0.005, 4, 6),
+	}, []string{"operation", "status_code"})
+
+	azureRequestDuration = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "tempodb",
+		Name:      "azure_request_duration_seconds",
+		Help:      "Time spent doing Azure requests.",
+		Buckets:   prometheus.ExponentialBuckets(0.005, 4, 6),
+	}, []string{"operation", "status_code"})
+
+	requestDuration = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
thank you for providing an upgrade path. let's note in the changelog that the old metrics are deprecated and this should be used instead.

Reply:
Done. Is it worth adding a note that the older metrics are deprecated in the metrics help?

Reply:
Agree. Let's add "DEPRECATED: See tempodb_backend_request_duration_seconds"

Reply:
Sounds good, that's done.